sonar-plugin-api/src/main/java/org/sonar/api/issue/filter/IssueFilterChain.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * SonarQube, open source software quality management tool.
+ * Copyright (C) 2008-2014 SonarSource
+ * mailto:contact AT sonarsource DOT com
+ *
+ * SonarQube is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * SonarQube is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.api.issue.filter;
```

Comment:
I'm not sure that this is the correct package to put this API. We have plan to maybe someday split batch and server apis so it is better if package reflect separation. And here it would make it clear that filtering is done on batch side (we could also have an issue filter on server side for "level 2" issues). I propose to start putting our APIs in org.sonar.api.scan (here it would be org.sonar.api.scan.issue.filter)